### PR TITLE
Fix Dell Optiplex variables to enable firmware download

### PR DESCRIPTION
**Pull request by lenzj**
I tried to run Dasharo DTS from a usb thumb drive using version v1.2.21 on a Dell Optiplex 9010 and was unsuccessful.  Others have reported this same issue (https://github.com/Dasharo/dasharo-issues/issues/706).

However after editing the dts-functions.sh script on the thumb drive with these included modifications I was able to complete the download and flash the firmware using DTS.

I was unable to determine what the original "BIOS_HASH_LINK_COMM" line was referring to and commented it out and things seemed to work OK.  Hopefully someone more knowledgeable can determine if my updates are correct and if that commented out line should be moved or modified somewhere else in the script.

By the way, thank you very much for your efforts on Dasharo BIOS.  Much appreciated.

### CODE REVIEW

**Comment by macpijan** [comment]
@tym2k1 will give it a test. Thanks for contribution @lenzj

File: meta-dts-distro/recipes-dts/dts/dts/dts-functions.sh [hunk 1]
```diff
@@ -488,7 +488,9 @@ board_config() {
       BIOS_LINK_COMM="$FW_STORE_URL/$DASHARO_REL_NAME/v$DASHARO_REL_VER.rom"
       HAVE_EC="false"
       NEED_EC_RESET="false"
-      BIOS_HASH_LINK_COMM="a880504dfb0497f31e898f62ae5c9f3b145bca0b4fa601e41cddc54bea22ee36  $BIOS_UPDATE_FILE"
+      #BIOS_HASH_LINK_COMM="a880504dfb0497f31e898f62ae5c9f3b145bca0b4fa601e41cddc54bea22ee36  $BIOS_UPDATE_FILE"
```
Comment (miczyg1) [inline meta-dts-distro/recipes-dts/dts/dts/dts-functions.sh:491]: We can simply remove that line, the hash is validated against the signature anyways.

Comment (miczyg1) [reply meta-dts-distro/recipes-dts/dts/dts/dts-functions.sh:491]:
```suggestion
```

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: meta-dts-distro/recipes-dts/dts/dts/dts-functions.sh (hunk 1 above)
Comment (lenzj) [reply meta-dts-distro/recipes-dts/dts/dts/dts-functions.sh]: Thanks for confirming.  I just pushed a commit to remove that commented line.

**Review by miczyg1 — APPROVED** [review]

**Comment by tym2k1** [comment]
I have verified the issue and confirmed that your changes resolve it.  I approve merging this PR.

Thank you for your efforts in improving Dasharo. Your contribution is greatly appreciated :)

**Tested on**: Dell Inc. Optiplex 7010
**Stock BIOS version**: A13
**HCL Report**: [link](https://cloud.3mdeb.com/index.php/s/6mHcN7NGT4fnApF)

**Logs before the fix**: [dts_logs_pre_fix.log](https://github.com/user-attachments/files/15740879/dts_logs_pre_fix.log)
**Logs after the fix**: [dts_logs_post_fix.log](https://github.com/user-attachments/files/15740880/dts_logs_post_fix.log)

**Review by tym2k1 — APPROVED** [review]

**Comment by macpijan** [comment]
@tym2k1 Please also apply to this repo: https://github.com/Dasharo/dts-scripts so we do not loose this change on transition.

**Comment by tym2k1** [comment]
[Migrated the commits and opened a new PR](https://github.com/Dasharo/dts-scripts/pull/12)

**Review by macpijan — APPROVED** [review]

**Pull request merged**